Automation/create_vm_script.ps1

```diff
@@ -0,0 +1,98 @@
+#This script creates a Ubuntu VM on Azure.
+#The script logs user into Azure using Service Principal. To log in,
+#Tenant ID, App ID, Secret, and Subscription ID are required.
+#User must provide Azure Resource Group Name, VM Name, Admin Username/Password
+
+param (
+    [string] [Parameter(Mandatory=$true)] $tenantId, #Tenant ID used to login to Azure with Service Principal
+    [string] [Parameter(Mandatory=$true)] $applicationId, #App ID used to login to Azure with Service Principal
+    [string] [Parameter(Mandatory=$true)] $secret, #Azure Secret used to login to Azure with Service Principal
+    [string] [Parameter(Mandatory=$true)] $subscriptionId, #Subscr. ID used to login to Azure with Service Principal
+    [string] [Parameter(Mandatory)] $rgName, #Resource Group Name required
```

Comment:
Is this different from saying Mandatory=$true? I still got prompted for these parameters, so maybe just add the -$true to make it look like the others